package.json

```diff
@@ -31,7 +31,7 @@
   "devDependencies": {
     "chai": "^2.3.0",
     "deep-extend": "^0.5.1",
-    "eslint": "^4.0.0",
+    "eslint": "^4.18.2",
     "eslint-config-loopback": "^10.0.0",
```

Comment:
Just curious, why is `eslint-config-loopback` here sets to `^10.0.0` while others are  `^8.0.0`? like [here](https://github.com/strongloop/loopback-bluemix/pull/26/files?file-filters%5B%5D=.json#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R38)

Reply:
We should update all packages to use the latest version of `eslint-config-loopback` (`^13.1.0` at the time of writing).